components/status-indicator/status-indicator.js

```diff
@@ -0,0 +1,99 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class StatusIndicator extends LitElement {
+
+	static get properties() {
+		return {
+			/**
+			 * State of status indicator to display. Valid values are: 'default', 'success', 'alert', and 'none'.
+			 */
+			state: {
+				type: String,
+				reflect: true
+			},
+			/**
+			 * Text for status indicator.
+			 */
+			text: {
+				type: String,
+				value: null
+			},
+			/**
+			 * Use when the status is very important and needs to have a lot of prominence.
+			 */
+			bold: {
+				type: Boolean,
+				value: false
+			}
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				border-color: var(--d2l-color-celestine);
+				border-radius: 0.6rem;
+				border-style: solid;
+				border-width: 1px;
+				color: var(--d2l-color-celestine);
+				cursor: default;
+				display: inline-block;
+				font-size: 0.6rem;
+				font-weight: bold;
+				line-height: 1;
+				overflow: hidden;
+				padding: 2px 10px 2px 10px;
+				text-transform: uppercase;
+				text-overflow: ellipsis;
+				vertical-align: middle;
+				white-space: nowrap;
+			}
+
+			:host([hidden]) {
+				display: none;
+			}
+			:host([state="success"]) {
+				border-color: var(--d2l-color-olivine-minus-1);
+				color: var(--d2l-color-olivine-minus-1);
+			}
+			:host([state="alert"]) {
+				border-color: var(--d2l-color-cinnabar);
+				color: var(--d2l-color-cinnabar);
+			}
+			:host([state="none"]),
+			:host([state="null"]) {

```

Comment:
I don't think we should have `state="null"`... ?

Reply:
I did consider removing it. I only left it to guarantee backwards compatibility. I'd be happy to remove it if we think it'd be safe to do so.

Reply:
It's there because it used to be a valid state and was removed. If we can somehow confirm that no one's using it, we could remove.